parlai/core/utils.py

```diff
@@ -1011,7 +1033,7 @@ def convert(key, value):
         if key not in ignore_fields.split(','):
             msg[key] = convert(key, value)
     msg['episode_done'] = msg.get('episode_done', False)
-    return msg
+    return Message(msg)
```

Comment:
What, does this handle it for FbDialogTeacher or ParlaiDialogTeacher? This is obviously correct, but I don't know who is using this...

Reply:
yeah parlaidialogteacher uses this